wazo_confd/plugins/endpoint_sip/schema.py

```diff
@@ -70,6 +72,60 @@ class EndpointSIPSchema(_BaseSIPSchema):
     line = fields.Nested('LineSchema', only=['id', 'links'], dump_only=True)
 
 
+class MergedEndpointSIPSchema(BaseSchema):
+    def get_attribute(self, obj, attr, default):
+        only = getattr(self.declared_fields[attr], 'only', None)
+        if attr.endswith('_section_options') and only:
+            options = get_value(obj, attr)
+            return [[key, value] for key, value in options if key in only]
+        return super().get_attribute(obj, attr, default)
+
+    uuid = fields.UUID(dump_only=True)
+    tenant_uuid = fields.UUID(dump_only=True)
+    name = fields.String(validate=PJSIPSection())
+    label = fields.String(validate=Length(max=128), allow_none=True)
+
+    combined_aor_section_options = options_field
+    combined_auth_section_options = options_field
+    combined_endpoint_section_options = options_field
+    combined_identify_section_options = options_field
+    combined_registration_section_options = options_field
+    combined_registration_outbound_auth_section_options = options_field
+    combined_outbound_auth_section_options = options_field
```

Comment:
Any reason to use a prefix instead of no prefix?
IMO if I request a merged view, I don't want to handle other key name. But it's only a preference, not a strong opinion

Reply:
I think I wrongly read this schema ! instead of doing a post_dump, you can use `attribute`